components/automate-ui/src/app/components/confirm-apply-stop-modal/confirm-apply-stop-modal.component.ts

```diff
@@ -46,7 +46,7 @@ export class ConfirmApplyStopModalComponent implements OnChanges {
     const durCountdown = etc.diff(now) < 0 ? '00:00:00' : `${durHours}:${durMins}:${durSecs}`;
 
     this.progressValue = percentageComplete;
-    this.progressPrefixText = `${Math.floor(percentageComplete * 100)}% complete`;
+    this.progressPrefixText = `${Math.floor(percentageComplete)}% complete`;
```

Comment:
Because the scale is now already normalized to 100 (in the reducer--so there is a single source of truth), this is duplicating the effort causing values to be too large.